resources/scripts/components/server/auditlogs/AuditLogHandler.ts

```diff
@@ -0,0 +1,95 @@
+import { ServerAuditLog } from '@/api/server/types';
+
+export function AuditLogHandler (log: ServerAuditLog) {
+    switch (log.action) {
+        case 'server:backup.start':
+            return `Created backup ${log.metadata.backup_name}`;
+        case 'server:backup.fail':
+            return `Backup ${log.metadata.backup_name} failed`;
+        case 'server:backup.complete':
+            return `Backup ${log.metadata.backup_name} completed`;
+        case 'server:backup.delete':
+            return `Deleted backup ${log.metadata.backup_name}`;
+        case 'server:backup.download':
+            return `Downloaded backup ${log.metadata.backup_name}`;
+        case 'server:backup.lock':
+            return `Locked backup ${log.metadata.backup_name}`;
+        case 'server:backup.unlock':
+            return `Unlocked backup ${log.metadata.backup_name}`;
+        case 'server:backup.restore.start':
+            return `Backup ${log.metadata.backup_name} restoration started`;
+        case 'server:backup.restore.complete':
+            return `Backup ${log.metadata.backup_name} restoration completed`;
+        case 'server:backup.restore.fail':
+            return `Backup ${log.metadata.backup_name} restoration failed`;
+
+        case 'server:database.create':
+            return `Created database ${log.metadata.database_name}`;
+        case 'server:database.password.rotate':
+            return `Rotated password of database ${log.metadata.database_name}`;
+        case 'server:database.delete':
+            return `Deleted database ${log.metadata.database_name}`;
+
+        case 'server:filesystem.download':
+            return `Downloaded file ${log.metadata.file}`;
+        case 'server:filesystem.write':
+            return `Edited file ${log.metadata.file}`;
+        case 'server:filesystem.delete':
+            if (log.metadata.files.length > 1) {
+                return `Deleted files ${log.metadata.files.join(', ')}`;
+            } else {
+                return `Deleted file ${log.metadata.files[0]}`;
+            }
+        case 'server:filesystem.rename':
+            return `Renamed file ${log.metadata.file}`;
```

Comment:
For this type, the actual metadata value is "log.metadata.files", which is an array on which objects have "from" and "to" values.

Not sure if there's any case in which the array could have more than a single object, which would mean needing to join them or loop over them, but otherwise you could get the value using "log.metadata.files[0].from" and "log.metadata.files[0].to" respectively.